src/main/kotlin/racingcar/controller/RacingGame.kt

```diff
@@ -1,27 +1,28 @@
-package racingcar.car
+package racingcar.controller
 
-import racingcar.car.exception.GameOverException
-import racingcar.car.strategy.MoveStrategy
-import racingcar.car.strategy.NameStrategy
+import racingcar.domain.Drivers
+import racingcar.domain.strategy.MoveStrategy
+import racingcar.domain.strategy.NameStrategy
+import racingcar.dto.MoveResults
+import racingcar.exception.GameOverException
 
 class RacingGame(carNames: List<String>, trialCount: Int, moveStrategy: MoveStrategy, nameStrategy: NameStrategy) {
     private val drivers: Drivers = Drivers(carNames, moveStrategy, nameStrategy)
+    private var _results: MutableList<MoveResults> = mutableListOf()
     private var _trialCount: Int = trialCount
 
     val trialCount get() = _trialCount
+    val results: List<MoveResults> get() = _results
```

Comment:
```suggestion
    val results: List<MoveResults> get() = _results.toList()
```
방어적 복사를 사용하시면 좋습니다! .toList()를 안해주면 results를 외부에서 mutable로 캐스팅 했을 때 내부에 숨겨둔 _results를 수정할 수 있어요.

Reply:
오 넵! toList 변경하겠습니다 :) 감사합니다